### PR TITLE
New Post: Error Code 137

### PR DESCRIPTION
**Pull request by AlbertPM9000**
Maybe it could be in a folder called Issues, inside Minecraft, where Korbs can also put the Invalid Bungeecord article

### CODE REVIEW

**Comment by itsjuoum** [comment]
I think there's no need to create a whole new article for this error as it can be easily sovlved without a lot of steps. Instead, it should be included in the [troubleshooting article](https://help.falixnodes.net/falix/general/troubleshooting/), but I'll let @KorbsStudio decide.

**Comment by AlbertPM9000** [comment]
> I think there's no need to create a whole new article for this error as it can be easily sovlved without a lot of steps. Instead, it should be included in the [troubleshooting article](https://help.falixnodes.net/falix/general/troubleshooting/), but I'll let @KorbsStudio decide.



> I think there's no need to create a whole new article for this error as it can be easily sovlved without a lot of steps. Instead, it should be included in the [troubleshooting article](https://help.falixnodes.net/falix/general/troubleshooting/), but I'll let @KorbsStudio decide.

Oh I didn't see that article. I think you're right

> I think there's no need to create a whole new article for this error as it can be easily sovlved without a lot of steps. Instead, it should be included in the [troubleshooting article](https://help.falixnodes.net/falix/general/troubleshooting/), but I'll let @KorbsStudio decide.

Oh I didn't see that article. I think you're right.

**Review by KorbsStudio — CHANGES_REQUESTED** [review]
An error does not deserve an entire article to it's self, but should be put into the error section of the Troubleshoot.md file: https://help.falixnodes.net/falix/general/troubleshooting/#errors

Everything must be done in one file as much as possible.

**Comment by KorbsStudio** [comment]
Also this is not for Minecraft, so it should be moved under General sub category in Falix.

**Comment by KorbsStudio** [comment]
Please also mention other information about making sure customers are using the proper version of Java for their selected version of Minecraft and also making sure that they have enough resources used for their server like the correct amount of CPU cores and memory as they can have an impact on the server if not setup properly.

or...I can do this myself, but I'm just giving you a chance.

File: docs/minecraft/general/error-code-137.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## How can I solve this?
+There are several ways to fix this.
+
+### Expand resources
+You can expand your resources (Cores and RAM) for your server to run less loaded. You're able to buy them from the store located in the [client panel](https://client.falixnodes.net). These resources are bought with Falixcoins (if you don't know how to get them, please visit this [article](https://help.falixnodes.net/falix/general/getting-coins)).
+
+<img src="https://cdn.discordapp.com/attachments/752954970558562354/857652571736899604/Resources.png">
+
+### Optimize server
```
Comment (KorbsStudio) [inline docs/minecraft/general/error-code-137.md]: Please also mention other information about making sure customers are using the proper version of Java for their selected version of Minecraft and also making sure that they have enough resources used for their server like the correct amount of CPU cores and memory as they can have an impact on the server if not setup properly.

or...I can do this myself, but I'm just giving you a chance.

Comment (AlbertPM9000) [reply docs/minecraft/general/error-code-137.md]: Do you mean in Optimizing article or error 137 one? I'm a little bit confused as it says you mentioned the Optimizing one but reviewed the 137.

Comment (KorbsStudio) [reply docs/minecraft/general/error-code-137.md]: Obviously optimizations, the error needs to go into the Troubleshooting article.

Comment (AlbertPM9000) [reply docs/minecraft/general/error-code-137.md]: Done. Can you see changes? I'm pretty new in GitHub, so I don't know if I have to enable something or do something.

Comment (KorbsStudio) [reply docs/minecraft/general/error-code-137.md]: I can move the stuff myself and get things to where they're suppose to be. 
So I'll take it from here.

**Comment by KorbsStudio** [comment]
Merged manually: https://github.com/FalixNodes-Software/help-center/commit/a2ec39c3a90f949c16973c96a66c26056b1b5517

**Pull request closed without merging**